web/server/codechecker_server/api/report_server.py

```diff
@@ -399,21 +417,31 @@ def get_other_source_component_file_query(session):
     """ Get filter query for the auto-generated Others component.
     If there are no user defined source components in the database this
     function will return with None.
+
+    The returned query will look like this:
+        (Files NOT IN Component_1) AND (Files NOT IN Component_2) ... AND
+        (Files NOT IN Component_N)
```

Comment:
```suggestion
        (Files NOT LIKE Component_1) AND (Files NOT LIKE Component_2) ... AND
        (Files NOT LIKE Component_N)
```

Reply:
With this I wanted to express that we will filter source files which are not `IN` the list of source files referenced by `Component_N`.